Languages/IronPython/IronPython.Modules/socket.cs

```diff
@@ -1297,6 +1297,10 @@ public int ioctl(BigInteger cmd, object option)
             )]
         public static string getfqdn(string host) {
             host = host.Trim();
```

Comment:
As an additional hint, you maybe could assert `host` for null and give a specific error message. Else it will crash on `.Trim()` with a very unlucky `NullReferenceException`.